src/nurse/tests/test_views.py

```diff
@@ -784,3 +784,55 @@ def test_post_only_staff(self, user, client):
         assert response.json() == {
             "detail": "You do not have permission to perform this action."
         }
+
+
+@pytest.fixture
+def user_profile(user):
+    return UserOneSignalProfile.objects.create(user=user, subscription_id="123456789")
+
+
+@pytest.fixture
+def another_user_profile(another_user):
+    return UserOneSignalProfile.objects.create(
+        user=another_user, subscription_id="987654321"
+    )
+
+
+@pytest.fixture
+def another_user(db):
+    return User.objects.create_user(
+        username="username2", email="user2@example.com", password="testpass123"
+    )
+
+
+@pytest.mark.django_db
+class TestUserOneSignalProfileView:
```

Comment:
Can we also have a test similar to `test_patient_list_401()` that basically show only authenticated list are allowed?

Reply:
I understood  really why patient here so we work on onssignal